R/dvhydrograph-render.R

```diff
@@ -249,7 +246,13 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", ...){
 
   x <- plotItem[['time']]
   y <- plotItem[['value']]
-  legend.name <- plotItem[['legend.name']]
+
+  if('legend.name' %in% names(plotItem)){
+    legend.name <- plotItem$legend.name
+  } else {
+    legend.name <- ""
+  }
```

Comment:
is there anyway to replace this with the `nullMask` function? 

`legend.name <- nullMask(plotItem$legend.name)`